### ISSUE TITLE
Services scripts should be rewritten using ansible.

### ISSUE THREAD

**Issue by gellner** [UKCloud/openshift-deployment-ansible#7]

**Comment by charliejllewellyn** [comment]
Please can you add more information to this so others can potentially pick it up

**Comment by stevemul** [comment]
@gellner I can't remember what this was about. Can you update/elaborate/close this issue when you get a chance please?

**Comment by gellner** [comment]
There's a few bash scripts in the ./services-tenancy directory which automate the deployment of rh-sso into a central services tenancy and creates lets-encrypt certs for it. There was a desire expressed to refactor them using ansible.

Since the plan was to have only once services tenancy per site, it didn't seem worth the effort. And, at the moment I doubt SSO is much of a priority now. So closed-wontfix?

**Comment by stevemul** [comment]
Ahh, yes that makes sense. I think you're probably correct too - this will depend on a central SSO service rather than the proposed services tenancy now, so I wouldn't spend too much time on refactoring it. It might even be worth stripping this work out and keeping it somewhere else so its not confusing things in here.

**Comment by gellner** [comment]
I have a feeling this isn't needed any longer....
